content/blog/pin-the-face-that-launches-a-thousand-ships.md

```diff
@@ -0,0 +1,142 @@
++++
+title = "Pin the Face that Launches a Thousand Ships"
+date = "2023-04-04"
+description = "A guest post by ~nospex-larsut"
+[extra]
+image = "https://storage.googleapis.com/media.urbit.org/blog/6a0120a85dcdae970b016766c94c14970b-800wi.jpg"
+author = ""
+ship = "~nospex-larsut"
++++
+
+![](https://storage.googleapis.com/media.urbit.org/blog/6a0120a85dcdae970b016766c94c14970b-800wi.jpg)
+
+## Pin the Face that Launches a Thousand Ships[^1] ##
+_A short piece addressing some of the standard hesitations to learning Hoon, considered in light of all of the recent exciting network activity._
+
+### Setting The Table ### 
+One can draw many distinctions between the Urbit apps that were available this time last year and those released thus far in 2023, but the most important of these may be that the former were apps noticed and used almost exclusively by users already on Urbit, where the latter - notably Pongo, Realm, Tirell Studio, Groups 2, Portal, etc - are apps that are _creating Urbit users_.   
+
+Prior to the start of 2023, the superficially yet incomplete take was something to the effect that much of Urbit’s appeal was largely an affinity with the ideas and principles surrounding the project.  
```

Comment:
"superficially yet incomplete" is missing a word, I think? Maybe superficially correct, yet incomplete?

Reply:
Go ahead and merge once you've decided what to do here. @thomasurbit